.github/workflows/docker.yaml

```diff
@@ -9,7 +9,7 @@ on:
   push:
     branches:
       - main
-      - release/*
+      - 'v*'
```

Comment:
This is triggering when we push a branch.

We could trigger this by [pushing git tag](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs) or by [creating/publishing Github release](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release)